api/src/migrations/api/1621963684141-ProjectSpecificWdpaEvents.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CostSurfaceEvents1621963684141 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      INSERT INTO api_event_kinds (id) values
+      ('project.wdpa.submitted/v1/alpha'),
+      ('project.wdpa.finished/v1/alpha'),
+      ('project.wdpa.failed/v1/alpha');
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `DELETE FROM api_event_kinds WHERE id like 'project.wdpa.%';`,
```

Comment:
as above if accepting the previous suggestion
```suggestion
      `DELETE FROM api_event_kinds WHERE id like 'project.protectedAreas.%';`,
```

Reply:
and so on for the rest of the PR 😱 - best to cross check with @aagm that she is indeed happy with this suggestion 😃